app/lib/three_scale/policies/specification.rb

```diff
@@ -6,16 +6,9 @@ class Specification
       extend ActiveModel::Naming
       extend ActiveModel::Translation
```

Comment:
We can also remove these 2 extends now 😄

Reply:
@hallelujah These 2 are not necessary anymore because these were here for the i18n of the error added `invalid JSON` that is not here anymore 😄